.github/workflows/ci.yaml

```diff
@@ -98,17 +98,14 @@ jobs:
         path: dist/*.whl
         if-no-files-found: error
 
-    - name: Install python dependencies
-      run: ./dev.py install-python-deps
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
 
-    - name: Install wheel packages
-      run: ./dev.py install-wheel-packages
-
-    - name: Install browsers
-      run: ./dev.py install-browsers
+    - name: Build docker images
+      run: ./dev.py e2e-build-images
 
     - name: Run e2e tests
-      run: ./dev.py run-e2e
+      run: ./dev.py e2e-run-tests
```

Comment:
We want to run E2E tests for only one version of NodeJS and one version of Python, but we should use different versions of Streamlit and The Streamlit Component Library.

Reply:
I will add this in https://snowflakecomputing.atlassian.net/browse/STREAMLIT-3898